app/src/main/java/waiter/Messenger.java

```diff
@@ -0,0 +1,18 @@
+package waiter;
+
+import java.io.IOException;
+
+public class Messenger implements Transporter {
+
+    Protocol server;
+
+    Messenger(Protocol server) {
```

Comment:
This is getting confusing to me. Somewhere in the code we're using `Awaiter server` and then we also have `ServerSocket server` and here we're using `Protocol server`. As I jump through the code I thought that all take in the same server but in reality they are completely different things. I would suggest calling them what they are, for example, `Protocol protocol`. Ultimately we want to make sure we' re using these terms in a consistent manner and avoid having one term means different things in different part of the codebase.

Reply:
Sounds good to me, I will have the names reflect what they are!

Reply:
Resolved in [this commit](https://github.com/jonmanahan/waiter/pull/2/commits/3325b9eee76dbc890d1836d2e1d440ec28adb18b), if it resolves the issues please hit resolve conversation if possible, thank you all a ton!